### ISSUE TITLE
Don't use the android download manager

### ISSUE THREAD

**Issue by VittGam** [jackpf/ApkDownloader#22]
Hello,

On 4.2.2 the android download manager fails, for both the play store and this app. Don't know why.  I think that this problem can be resolved by implementing the downloader part in-app, like the firefox integrated downloader is.

Thanks!
Vittorio G

**Comment by jackpf** [comment]
I've just tested it on a Nexus 5 emulator running 4.2.2 and it seems to work fine. Could you be a bit more specific about what device you're on and what is actually failing?

Cheers,
Jack

**Comment by h00d3d** [comment]
yeah please build in-app downloader. i even cant see apk's size with default download manager. And one thing, could you do the list of devices/countries to choose from . thanks for your great app.
Samsung Galaxy S2 (i9100)
